hal/stm32/pwm_stm32.cpp

```diff
@@ -198,7 +198,7 @@ bool Pwm_stm32::init(void)
 
 bool Pwm_stm32::set_pulse_width_us(uint16_t pulse_us)
 {
-    duty_cyle_ = pulse_us;
+    duty_cycle_ = pulse_us;
```

Comment:
In the code, duty cycle is used as an equivalent to pulse width as it is in the datasheet of the stm32. The register used to store the pulse width is named duty cycle and goes from 0 to period of pulse.